src/Interactive/HostProcess/InteractiveHost64.csproj

```diff
@@ -11,7 +11,7 @@
     <ExcludeFromSourceBuild>true</ExcludeFromSourceBuild>
 
     <!-- Publishing (only precompile binaries when building on CI to avoid slowing down dev builds by ~10s) -->
-    <PublishReadyToRun Condition="'$(ContinuousIntegrationBuild)' == 'true'">true</PublishReadyToRun>
+    <PublishReadyToRun Condition="'$(ContinuousIntegrationBuild)' == 'true' AND '$(NoPublishReadyToRun)' != 'true'">true</PublishReadyToRun>
```

Comment:
>PublishReadyToRun [](http://example.com/codeflow?start=5&length=17)

Set `/p:PublishReadyToRun=false` instead.

Reply:
Is this actually necessary to disable explicitly? Should only be triggered by running `Publish` target.

Reply:
@genlu Hmm, might be triggered by VSIX targets. If it's not related to Publish target we should probably rename it.